libs/components/src/lib/accordion-item/accordion-item.template.ts

```diff
@@ -0,0 +1,61 @@
+import { html, when } from '@microsoft/fast-element';
+import type { ViewTemplate } from '@microsoft/fast-element';
+import type {
+	ElementDefinitionContext,
+	FoundationElementDefinition,
+} from '@microsoft/fast-foundation';
+import { classNames } from '@microsoft/fast-web-utilities';
+import { affixIconTemplateFactory } from '../../shared/patterns/affix';
+import type { AccordionItem } from './accordion-item';
+
+const getClasses = ({
+	open, iconTrailing, icon, dense, noIndicator
+}: AccordionItem) => classNames(
+	'control',
+	['open', open],
+	['icon', Boolean(icon)],
+	['icon-trailing', iconTrailing],
+	['dense', dense],
+	['no-indicator', noIndicator],
+);
+
+export const AccordionItemTemplate: (
+	context: ElementDefinitionContext,
+	definition: FoundationElementDefinition
+) => ViewTemplate<AccordionItem> = (context) => html<AccordionItem>`
+	<div class="${getClasses}">
+		${x => renderPanelHeader(context, x.headingLevel)}
+		<div class="accordion-item-body" id="content" role="region" aria-labelledby="header">
```

Comment:
css convention of scoped portions was needed in large scale style files to namespace parts and avoid conflicts
this is redundant for web components and might even deviate from the web components templates convention. templates has their own encapsulated tree

prefixes are no longer required

```suggestion
		<div class="body" id="content" role="region" aria-labelledby="header">
```

Reply:
let's remove it from other classes?